class-pinterest-for-woocommerce.php

```diff
@@ -639,6 +640,26 @@ public static function disconnect() {
 		}
 
 
+		/**
+		 * Flush data option and remove settings.
+		 *
+		 * @return void
+		 */
+		private static function clean_options_on_disconnect() {
```

Comment:
1 minor thing, I would call this method something not specific to disconnect. Eg `clean_options` or `flush_options`, since what it does aren't necesserily tied to disconnect.